### ISSUE TITLE
error in process filter: Wrong type argument: sequencep, :name 

### ISSUE THREAD

**Issue by thorstengrothe** [lordpretzel/mu4e-views#30]
Hi,

I installed mu4e-views this way

```
(use-package mu4e-views
  :straight (mu4e-views :type git :host github :repo "lordpretzel/mu4e-views" :branch "mu-1.7-support")
(...)
```

on

`Emacs 29.0.50 (build 1, x86_64-pc-linux-gnu, GTK+ Version 3.24.34, cairo`

Xwidgets is running fine here

with

`mu 1.8.7`

When I try to open a html-message I got this error

`error in process filter: Wrong type argument: sequencep, :name `

I have absolutely no idea what I can do?

Can you anyone help me?

Thanks in advance
Thorsten

**Comment by lordpretzel** [comment]
Hi Thorsten,

several API method names and data structures changed between 1.7 and the 1.8 release. I have not gotten around to fix these yet and realistically I will not get to  work on this the next two weeks.

Best,

Boris

**Comment by thorstengrothe** [comment]
Hi Boris,

no problem, take your time!

Would it help to downgrade mu4e to 1.6 or 1.7?

Regards
Thorsten

**Comment by lordpretzel** [comment]
yes, the main branch works with 1.6. I am using the branch you tried out as my daily driver for 1.7 (there are some bugs, but it is usable)

**Comment by thorstengrothe** [comment]
Ok, tried to install mu4e 1.6 with straight-use-package but it does not work, got a lot of errors. Normally I install with my package manager (Archlinux here) but this works only for the 1.8 series, I cannot find a archlinux package for the 1.6/1.7 series. How do you install mu4e on your system and how do you integrate this with straight? There are some peoples out there with similar problems. See here:

[Install mu4e with straight](https://github.com/radian-software/straight.el/issues/491)

Thanks in advance!
Thorsten

**Comment by lordpretzel** [comment]
Hi Thorsten,

straight would only install mu4e, but you would still be missing the `mu` binary with the right version. Until 1.8, I used 1.7 and installed it from source. The good news is that I've gotten around to fix some of the problems with v1.8 support in mu4e-views. There are some bugs (see #26), but it is usable. If you want to give it a try it is available in the `mu-1.8-support` branch. However, until the bigger bugs are fixed, I won't be merging it into master yet.

Cheers,

Boris

**Comment by thorstengrothe** [comment]
Hi Boris,

good to hear that! I will try the `mu-1.8-support` branch and see if it works for me. Thanks for your answer and all the work!

Regards
Thorsten
